server/src/routes/clientRouter.js

```diff
@@ -3,7 +3,7 @@ import getClientIndexPath from "../config/getClientIndexPath.js";
 
 const router = new express.Router();
 
-const clientRoutes = ["/", "/user-sessions/new", "/users/new"];
+const clientRoutes = ["/", "/user-sessions/new", "/users/new", "/shows/:id", "/shows/:id/info", "/shows"];
```

Comment:
not all of these new paths are being used. only the second to last one is. Only add paths for features that exist